docs/js/dynamics.js

```diff
@@ -0,0 +1,113 @@
+const comedians = [
+  {
+    name: 'MC Kisoli',
+    biography: "Bachelor's Degree, Government Worker, Lecturer",
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.Maiores error blanditiis necessitatibus soluta. Eveniet, quae.',
+    imgSrc: './images/mtubaki5.jpg',
+    imgAlt: 'Mtubaki1',
+  },
+  {
+    name: 'Joe Master',
+    biography: "Bachelor's Degree, Government Worker, Lecturer",
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.Maiores error blanditiis necessitatibus soluta. Eveniet, quae.',
+    imgSrc: './images/mtubaki2.jpg',
+    imgAlt: 'Mtubaki2',
+  },
+  {
+    name: 'Mtoto wa Dada',
+    biography: 'Business Owner',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.Maiores error blanditiis necessitatibus soluta. Eveniet, quae.',
+    imgSrc: './images/mtubaki4.jpg',
+    imgAlt: 'Mtubaki3',
+  },
+  {
+    name: 'Dogo Pepe',
+    biography: "Bachelor's Degree, Private Sector Worker",
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.Maiores error blanditiis necessitatibus soluta. Eveniet, quae.',
+    imgSrc: './images/mtubaki1.jpg',
+    imgAlt: 'Mtubaki4',
+  },
+  {
+    name: 'Kipatoshi',
+    biography: "Bachelor's Degree, Government Worker, Lecturer",
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.Maiores error blanditiis necessitatibus soluta. Eveniet, quae.',
+    imgSrc: './images/mtubaki3.jpg',
+    imgAlt: 'Mtubaki5',
+  },
+];
+
+const comediansCards = document.querySelector('#comedians-cards');
+let comediansNumbertoShow = 2;
+let nextStartIdx = 0;
+
+// Takes number of comedians to show, and where
+// to start in list and adds card ?
+// markup for each comedian in the list
```

Comment:
- [ ] You can remove the comments from the code, to make it more clear and professional.

Reply:
![image](https://user-images.githubusercontent.com/22321030/152308255-9a849b15-d878-4f34-bdff-2f68dfebf2b2.png)
I do not know what heppened but the social links are there

Reply:
I asked this question to the previous reviewer, by doing so, will the code be better?

Reply:
Yes, the code will be more readable, cleaner, and more professional 😉. Thanks for working on it anyways 👍

Reply:
So you mean, I should not include comments in any code? I thought it was part of the documentation of the design for better maintainability and eventually professionalism.

Reply:
Having comments on an obvious code behavior is not that good practice. Your code is self-explanatory. Of course, having commented code on complicated code behavior is not a bad idea if you're using them as instruction ✌️

Reply:
👍

Reply:
might be obvious to someone but not very so to someone else, don't you think? Any ways, thanks